### ISSUE TITLE
[Effect Suggestion] SCP-173

### ISSUE THREAD

**Issue by SpacyRainbow** [gta-chaos-mod/ChaosModV#2241]
**Name of effect**
Peanut
SCP-173

**Duration if timed effect (long or short timed?)**
Short or medium effect time 

**Effect Description**
You get put in first person view
An NPC spawns with extreme speed, it can be t posing, when looked at it does not move, but when not looked at, it will run extremely fast towards the player to snap their neck (kill them) if the player runs around a corner, the NPC can run around the corner at extreme speed till seen again

Tl DR when looked at, does nothing, when not observed it will attempt to kill the player
